pull request

saisumanthkacham opened the pull request.

saisumanthkacham committed:
setting up the page

saisumanthkacham committed:
styled header and navbar

color scheme implemented in nav bar and header

saisumanthkacham committed:
avatar block is updated

saisumanthkacham committed:
Alert's html and style are updated

saisumanthkacham committed:
badge and avatar components updated

wrote styling and html of badge and avtar-icon

saisumanthkacham committed:
button component updated

html and css of button component updated

saisumanthkacham committed:
button and card components partially updated

button - small and md components introducedd
 card- just skeleton with img updated

saisumanthkacham committed:
card  updated and small code refactoring

card- css , badge, and some html updated
converted div's--> item tag

saisumanthkacham committed:
code refactoring

saisumanthkacham committed:
refactored the ui elements

changed the images in cards and  changed the badges

saisumanthkacham committed:
cards updated

added horizontal card and text-card

saisumanthkacham committed:
added modal and recatored few ui elements

saisumanthkacham committed:
updated hover effects and ui changes

saisumanthkacham committed:
added iframes

saisumanthkacham committed:
added definitions for each element

saisumanthkacham committed:
added getting started page and updated hover properties to a link

saisumanthkacham committed:
added final iframe

saisumanthkacham committed:
para refactor

saisumanthkacham committed:
renamed the two html pages

saisumanthkacham committed:
adjusted the height and widths

sakshambhatt commented:
Hi @saisumanthkacham , everything looks good. I just had a question, why did you feel the need to add `flex-direction: row` to your flex(s), if by default their direction is row*?

harshitbadollacodes commented on index.html:
```diff
@@ -0,0 +1,52 @@
+<body class="grey-bg">
+
+    <div class="page-align">
+        <item class="navb white-font margin-zero">
```
Use semantic html. So instead of item, use <nav></nav>

saisumanthkacham replied:
fixed this issue with "section" and "div" tags  as they are part of semantic HTML

harshitbadollacodes replied:
Since this is navbar, use nav tag "<nav> </nav>". That is more apt than section and div

saisumanthkacham replied:
Done.

harshitbadollacodes commented on style.css:
```diff
@@ -0,0 +1,205 @@
+    color: var(--green-color);
+}
+
+.orange-bg {
```
Assign this color also in a varibale .

saisumanthkacham replied:
fixed this one :)

saisumanthkacham committed:
removed "flex-direction: row"

it is an unnecessary addition, because by default flex direction is row

saisumanthkacham commented:
> Hi @saisumanthkacham , everything looks good. I just had a question, why did you feel the need to add `flex-direction: row` to your flex(s), if by default their direction is column?

ya fixed this @sakshambhatt  , thankyou for pointing this out :)

Ankit-Nag reviewed (commented):
Your library looks great Sai! A few notes-

1. As mentioned by Harshit, use semantic HTML, i.e, avoid using <item>
2. Add a hover state to your primary button with fill. Since the background colour is already a primary colour, it's not changing state.

saisumanthkacham committed:
ui and code refactoring

fixed hovering of primary button
fixed the variable naming of color

saisumanthkacham committed:
buttons pointer updated

saisumanthkacham commented:
> Your library looks great Sai! A few notes-
> 
> 1. As mentioned by Harshit, use semantic HTML, i.e, avoid using
> 2. Add a hover state to your primary button with fill. Since the background colour is already a primary colour, it's not changing state.

1.fixed this issue with "section" and "div" tags  as they are part of semantic HTML
2.fixed hovering of primary button

saisumanthkacham committed:
code refactor

replaced the <item> tag with semantic HTML tags <div> and <section>

saisumanthkacham commented:
> Your library looks great Sai! A few notes-
> 
> 1. As mentioned by Harshit, use semantic HTML, i.e, avoid using
> 2. Add a hover state to your primary button with fill. Since the background colour is already a primary colour, it's not changing state.

prerana1821 commented on index.html:
```diff
@@ -0,0 +1,54 @@
+    <div class="page-align">
+        <item class="navb white-font margin-zero">
+            <img src="images/dunzo logo2.svg" class="navb-logo">
+            <ul class="navb-li">
```
Optional: a clearer name for class can be used for user friendliness as navbar or navigation-bar

prerana1821 commented on index.html:
```diff
@@ -0,0 +1,54 @@
+            <br><br>
+
+            <iframe src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=text%2Fx-c%2B%2Bsrc&ds=false&dsyoff=25px&dsblur=68px&wc=false&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Fgithub.com%252Fsaisumanthkacham%252FDunzo-UI%252Fblob%252Fd ...
+                style="width: 1024px; height: 473px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
```
The common styling in iframe component can be moved to a class as it's getting reused.

prerana1821 commented on mainDoc.html:
```diff
@@ -0,0 +1,544 @@
+
+    <nav class="nav ">
+        <div>
+            <img src="images/dunzo logo2.svg" class="logo">
```
use alt as much as possible as alt attribute provides alternative information for an image

prerana1821 commented on mainDoc.html:
```diff
@@ -0,0 +1,544 @@
+                <img src="images/an.svg" alt="" class="ava ava-md">
+
+                <br><br>
+                <iframe src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=lucario&wt=none&l=css&ds=false&dsyoff=20px&dsblur=68px&wc=false&wa=true&pv=0px&ph=0px&ln=true&fl=1&fm=Hack&fs=13px&lh=139%25&si=false&es=2x&wm=false&code=%2520%253Cdiv%253E%250A%2520%2520%253Cimg%2520src%253D%2522images%252Fgirl.svg%2522%2520alt%253D%2522%2522%2520class%253D%2522ava%2520ava-md%2520%2522 ...
```
small change: this can be formatted correctly!

prerana1821 commented on mainDoc.html:
```diff
@@ -0,0 +1,544 @@
+        <div class="avatar " id="avatar">
+
+            <h2>⚡Avatars</h2><br>
+            <p class="des-txt">In computing, an avatar (also known as a profile picture or userpic) is a graphical representation of a user or the user's character or persona</p>
```
a clearer name for class can be used here as well

prerana1821 commented on mainDoc.html:
```diff
@@ -0,0 +1,544 @@
+            <section>
+                <h3>>> Icon Avatar</h3><br>
+                <div class="row ">
+                    <div class="icon ter-bg "><i class="fab fa-snapchat-ghost fa-2x white-font"></i>
```
```suggestion
                    <div class="icon tertiary-bg "><i class="fab fa-snapchat-ghost fa-2x white-font"></i>
```
Use the same naming convention

prerana1821 commented on mainDoc.html:
```diff
@@ -0,0 +1,544 @@
+            <div class="row">
+                <h3> >> Badge On Cards </h3>
+                <div>
+                    <item class="primary-bg cd-bdg white-font">Trending</item>
```
```suggestion
                    <item class="primary-bg bdg-card white-font">Trending</item>
```
As you are using bdg as a prefix so you can follow the same naming convention.

prerana1821 commented on style.css:
```diff
@@ -0,0 +1,204 @@
+    padding: 0;
+    font-family: 'Lato', sans-serif;
+    /* font-family: 'Open Sans', sans-serif; */
+    /* font-family: 'Montserrat', sans-serif; */
```
commented code can be removed.

saisumanthkacham committed:
code refactor

div to nav

Ankit-Nag commented on style.css:
```diff
@@ -0,0 +1,204 @@
+    margin: 0;
+    padding: 0;
+    font-family: 'Lato', sans-serif;
+    /* font-family: 'Open Sans', sans-serif; */
+    /* font-family: 'Montserrat', sans-serif; */
```
Delete code that you won't be using

saisumanthkacham replied:
done

saisumanthkacham committed:
code refactor

deleted the unwanted code for fonts

prerana1821 commented on gettingStarted.css:
```diff
@@ -0,0 +1,123 @@
+    height: 20rem;
+}
+
+footer {
+    width: 100%;
+    text-align: center;
+    background-color: black;
+    color: white;
+    padding: 1rem 0;
+}
```
```suggestion
footer {
    width: 100%;
    text-align: center;
    background-color: black;
    color: white;
    padding: 1rem 0;
    position: fixed;
    bottom: 0;
}
```
This will make the footer fixed to bottom

prerana1821 reviewed (commented):
Hey @saisumanthkacham,

Your code base is really good, I liked the color palatte you have used, I have suggested some changes, try to be consistent with naming & remove unwanted code. Also It would be more beneficial if you separate out your code wrt components you have.

saisumanthkacham committed:
ecom

saisumanthkacham committed:
code: refactor

saisumanthkacham committed:
code: refactor

saisumanthkacham committed:
style: margin added to card

saisumanthkacham committed:
style : refactor

saisumanthkacham committed:
style: image position added

saisumanthkacham committed:
style: card style updated

saisumanthkacham committed:
styles updated

edited according to creators cart requirements

saisumanthkacham committed:
style: card height updated

saisumanthkacham committed:
style : update btn hover property

saisumanthkacham committed:
style: color prop refactor

saisumanthkacham committed:
style: delete button

saisumanthkacham committed:
style: applied logo font bold

saisumanthkacham committed:
style refactor

saisumanthkacham committed:
refactor

saisumanthkacham committed:
refactor

saisumanthkacham committed:
refactor

saisumanthkacham committed:
refactor
